docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -78,6 +78,8 @@ There you can find links to upgrade notes for other versions too.
         -        $registrationPage->seeEmailError('Email no-reply@shopsys.com is already registered');
         +        $registrationPage->seeEmailError('This e-mail is already registered');
         ```
+- `PaymentFormType` fields `vat` and `czkRounding` were moved from `basicInformation` to `prices` group ([#956](https://github.com/shopsys/shopsys/pull/956))
```

Comment:
change it please into instructions for developer.
for instance like:
- change selection of fileds from extended froms ([#956](https://github.com/shopsys/shopsys/pull/956))
    - `PaymentFormType` fields `vat` and `czkRounding` from `basicInformation` to `prices` group
    - `TransportFormType` field `vat` from `basicInformation` to `prices` group